QSEEComFunc.c

```diff
@@ -72,8 +72,8 @@ int32_t qsee_open_handle(struct qsee_handle_t **ret_handle) {
 
     ALOGD("Loaded QSEECom API library...\n");
     // Setup internal functions
-    handle->ion_alloc = qcom_km_ion_memalloc;
-    handle->ion_free = qcom_km_ion_dealloc;
+    // handle->ion_alloc = qcom_km_ion_memalloc;
+    // handle->ion_free = qcom_km_ion_dealloc;
```

Comment:
remove

Reply:
Neither of these can be removed (or we remove all [legacy] FPC support!), since we still have Tama on A13? At least I cannot find it in https://developer.sony.com/develop/open-devices/downloads/software-binaries, maybe we should remove this legacy cruft now!

Reply:
we will do a T branch

Reply:
Since we "decided to keep supporting" Tama on A13 (or at the very least not drop this outright), we just need a branch for Android 12 where `libdmabufheap` was first introduced, if https://source.android.com/docs/core/architecture/kernel/dma-buf-heaps is believed. I'll port the legacy code to this library as well.